v2/blockstore/readonly_test.go

```diff
@@ -269,3 +271,46 @@ func TestReadOnlyErrorAfterClose(t *testing.T) {
 	// TODO: test that closing blocks if an AllKeysChan operation is
 	// in progress.
 }
+
+func TestNewReadOnly_CarV1WithoutIndexWorksAsExpected(t *testing.T) {
+	carv1Payload := `38a265726f6f747381d82a5823001220da8ca6ffee4c4cab3d2d5aea4c3320a1100f593c03ce9061f6dbc9bb6de3f36b6776657273696f6e01c1011220da8ca6ffee4c4cab3d2d5aea4c3320a1100f593c03ce9061f6dbc9bb6de3f36b122b0a2212204ccb05f719de64d9754e6d044a05c65f6043b872f8fde2fadaf68d80c9b4442e120018aed4e015122b0a2212209a4038931ac6e4e5794dc1efa1005c14048bfd4e533ff4ad499d172bea53ed74120018aed4e015122b0a221220bd4cb50acf4908a46cc11de1591cd567d2a00aedacad744757f969ed1aeb31a81200188ce5fb020a16080218b3d9bb2e208080e015208080e01520b3d9fb02`
```

Comment:
I wonder if you could reuse one of our existing sample CAR files in testdata, to avoid adding more "blobs" to the test code.